app/Http/Controllers/Maps/CustomMapDataController.php

```diff
@@ -294,7 +294,7 @@ public function save(Request $request, CustomMap $map): JsonResponse
 
     private function rateString(int $rate): string
     {
-        return Number::formatSi($rate, 0, 0, 'bps');
+        return Number::formatSi($rate, 2, 3, 'bps');
```

Comment:
You could have used "null" here to get the default (and the default is indeed 2 and 3).